### ISSUE TITLE
BUG in select2.js  MultiSelect2 

### ISSUE THREAD

**Issue by dimonixx** [orocrm/platform#356]
```javascript
this.opts.initSelection(this.opts.element, function(data){
                    var ids=$.map(data, self.id);
                    self.setVal(ids);
                    self.updateSelection(data);
                    self.clearSearch();
                    if (triggerChange) {
                        self.triggerChange(this.buildChangeDetails(oldData, this.data()));
                    }
});
```
Line 2947:
In the line self.triggerChange(this.buildChangeDetails(oldData, this.data()));  this is undefined. 
In the newer version of Select2 it's already fixed.

**Comment by Grygir** [comment]
Thanks to reporting the issue. It will be fixed in coming release.

**Comment by artml** [comment]
Fixed in 1.9 release. Thanks for reporting this!
